git/internal/e2e/run.sh

```diff
@@ -20,6 +17,4 @@ if [[ "${GO_TEST_PREFIX}" = "" ]] || [[ "${GO_TEST_PREFIX}" = *"TestGitLabCEE2E"
 fi
 
 cd "${DIR}"
-CGO_LDFLAGS=$(PKG_CONFIG_PATH="${PKG_CONFIG_PATH}" pkg-config --libs --static --cflags libgit2)
-PKG_CONFIG_PATH="${PKG_CONFIG_PATH}" CGO_LDFLAGS="${CGO_LDFLAGS}" CGO_ENABLED=1 \
-    go test -v -tags 'netgo,osusergo,static_build,e2e' -race -run "^${GO_TEST_PREFIX}.*" ./...
+CGO_ENABLED=1 go test -v -tags 'netgo,osusergo,static_build,e2e' -race -run "^${GO_TEST_PREFIX}.*" ./...
```

Comment:
```suggestion
CGO_ENABLED=0 go test -v -tags 'netgo,osusergo,static_build,e2e' -race -run "^${GO_TEST_PREFIX}.*" ./...
```

Reply:
We cgo for sha collision detection right?

Reply:
no we don't need it for sha collision detection but we do need it for `-race` so please ignore this suggestion